js/dialog.js

```diff
@@ -48,3 +66,30 @@ export function updateDetailsDialog(data, cardUrl) {
   `
   );
 }
+
+function animateCard(dialogCard){
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].classList.add("animated", "unclickable");
+}
+function resetCardEffects(){
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].classList.remove("holo", "basic", "secret", "rare", "ultra", "animated", "unclickable");
+}
+
+export function setCardRarity(folder){
```

Comment:
(nit) purely a code organization thing - this function makes more sense to be placed in cards.js
optional/ can be addressed later

Reply:
Makes sense! I have updated this. :)